src/cmd/linuxkit/pkglib/pkglib.go

```diff
@@ -169,6 +172,36 @@ func NewFromCLI(fs *flag.FlagSet, args ...string) (Pkg, error) {
 		}
 	})
 
+	var srcHashes string
+	sources := []string{pkgPath + ":/"}
+
+	for _, source := range pi.ExtraSources {
+		tmp := strings.Split(source, ":")
```

Comment:
You do this splitting and what have you in two places, you could consider making sources be a `[]struct Foo {src, dst string}` perhaps and accumulating it once here.